### ISSUE TITLE
Quickstart Typo

### ISSUE THREAD

**Issue by johangithub** [dusktreader/flask-praetorian#54]
Hello, I'm going through the quickstart guide for the first time to give flask-praetorian a try. In the quickstart, 
```
POST /auth HTTP/1.1
Host: localhost:5000
Content-Type: application/json
{
    "username": "TheDude",
    "password": "abides"
}
```
should be 
```
POST /login
```

for the current example as it does not have /auth endpoint. The docs should also reflect that.

**Comment by dusktreader** [comment]
Thanks for pointing that out. I'll get the documentation updated soon.

**Comment by dusktreader** [comment]
Resolved by PR #56
